controllers/apply/awards-for-all/fields.js

```diff
@@ -2057,6 +2057,38 @@ module.exports = function fieldsFor({ locale, data = {} }) {
                 }
             ]
         },
+        mainContactIsValid: {
+            name: 'mainContactIsValid',
+            label: localise({
+                en: `I confirm that the main and senior contacts aren't married or in a long-term relationship with each other, living together at the same address, or related by blood`,
+                cy: ''
+            }),
+            type: 'checkbox',
+            options: [
+                {
+                    value: 'yes',
+                    label: localise({
+                        en: 'Yes',
+                        cy: ''
+                    })
+                }
+            ],
+            isRequired: true,
+            get schema() {
+                return multiChoice(this.options).required();
```

Comment:
I'd try adding a `.strip()` to this as we don't really need/want this in the resulting schema, just for the validations.

Reply:
This prevents submission so we can't do this!

Reply:
Not sure I follow?